### PR TITLE
Fixed code style

### PR DESCRIPTION
**Pull request by revolter**
Extracted duplicate "hack" into a clearly named function. Previously, not all
instances had the explanation comment, and its side effect was not clear.

### CODE REVIEW

**Review by hebertialmeida — APPROVED** [review]
Much better, thanks...

File: Source/FolioReaderWebView.swift [hunk 1]
```diff
@@ -273,7 +268,15 @@ open class FolioReaderWebView: UIWebView {
...
 	}
 
-	// MARK: WebView direction config
+	// MARK: WebView
+
+	func clearTextSelection() {
+		// Forces text selection clearing
+		// @NOTE: this doesn't seem to always work
+
+		self.isUserInteractionEnabled = false
```
Comment (hebertialmeida) [inline Source/FolioReaderWebView.swift:277]: Don't need to use `self` here.

File: Source/FolioReaderWebView.swift [hunk 2]
```diff
@@ -129,8 +127,8 @@ open class FolioReaderWebView: UIWebView {
 		let selectedText = js("getSelectedText()")
 
 		setMenuVisible(false)
-		isUserInteractionEnabled = false
-		isUserInteractionEnabled = true
+
+		self.clearTextSelection()
```
Comment (hebertialmeida) [inline Source/FolioReaderWebView.swift:131]: Don't need to use `self` here.

**Review by hebertialmeida — APPROVED** [review]

File: Source/FolioReaderWebView.swift (hunk 2 above)
Comment (revolter) [reply Source/FolioReaderWebView.swift:131]: I know, but it's more consistent and clear this way.

Comment (revolter) [reply Source/FolioReaderWebView.swift:131]: I am personally going by [this answer][1]:

> I would prefer to keep using self whenever I'm using a property to omit these misunderstandings.

What do you think?

  [1]: http://stackoverflow.com/a/40368514/865175

**Pull request merged**